assets/src/blocks/Cookies/CookiesFieldResetButton.js

```diff
@@ -0,0 +1,27 @@
+const { __ } = wp.i18n;
+
+import { Tooltip } from '@wordpress/components';
+
+const COOKIES_DEFAULT_COPY = window.p4bk_vars.cookies_default_copy || {};
+
+export const CookiesFieldResetButton = ({ fieldName, toAttribute, currentValue }) => {
+  const defaultValue = COOKIES_DEFAULT_COPY[fieldName] || '';
+
+  if (!currentValue || !defaultValue || currentValue === defaultValue) {
+    return null;
+  }
+
+  return (
+    <div className='field-reset-button'>
+      <span
+        className='cta'
+        onClick={() => toAttribute(fieldName)(defaultValue)}
```

Comment:
I think we actually need to unset the attribute here. Otherwise it will not change along when the option changes.

Reply:
Updated! I've used `undefined` and it seems to work as expected 🙂